bakta/io/gff.py

```diff
@@ -193,6 +193,8 @@ def write_gff3(genome: dict, features_by_contig: Dict[str, dict], gff3_path: Pat
                         'product': feat['product'],
                         'Dbxref': feat['db_xrefs']
                     }
+                    if(feat.get(bc.PSEUDOGENE, None)):
+                        annotations[bc.INSDC_FEATURE_PSEUDOGENE] = feat['pseudogene']['qualifier']
```

Comment:
Please use the proper feature key constant (`constants.py/PSEUDOGENE`) instead of the plain string literal